SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -31,36 +34,36 @@ void FSpatialTestSettings::Override(const FString& MapName)
 	Duplicate(OriginalSpatialGDKEditorSettings);
 	Duplicate(OriginalGeneralProjectSettings);
 	Duplicate(OriginalEditorPerformanceSettings);
-	// First override the settings from the base config file, if it exists
+
+	// Base config, applies to all maps if present
 	if (FPaths::FileExists(BaseOverridesFilename))
 	{
+		// Override the settings from the base config file
 		Load(BaseOverridesFilename);
 	}
-	else
-	{
-		UE_LOG(LogSpatialTestSettings, Log, TEXT("Base config file not found %s."), *BaseOverridesFilename);
-	}
-	// Then override the settings from the map specific config file, if it exists
-	FString MapOverridesFilename = OverrideSettingsBaseFilename + FPackageName::GetShortName(MapName) + (OverrideSettingsFileExtension);
```

Comment:
With the removal of the map overrides, do we have anything that needs updating in the TestGyms?

Reply:
Yes I need to update the TestGyms in v4.25 so I will do that now I have approval on the approach.